scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -386,6 +352,8 @@ static bool refresh_cpumasks(u32 layer_id)
 				cpuc->layer_id = layer_id;
 				bpf_cpumask_set_cpu(cpu, layer_cpumask);
 				total++;
+
+				scx_bpf_kick_cpu(cpu, SCX_KICK_IDLE);
```

Comment:
Silly question, what does the kick idle do in this path?

Reply:
e.g. Let's say a system has a confined layer which is currently the only active one. A lot of tasks woke up to the layer just now and the layer expanded quite a bit. Let's say that the tasks don't sleep much - they just wake up, run for a long time and then go back to sleep. Without the above kick, when the layer expands with those tasks queued, the new CPUs will sit idle until some tasks wake up to them.